src/include/rbd/features.h

```diff
@@ -76,15 +82,17 @@
 #define RBD_FEATURES_SINGLE_CLIENT (RBD_FEATURE_EXCLUSIVE_LOCK | \
                                     RBD_FEATURE_OBJECT_MAP     | \
                                     RBD_FEATURE_FAST_DIFF      | \
-                                    RBD_FEATURE_JOURNALING)
+                                    RBD_FEATURE_JOURNALING     | \
+                                    RBD_FEATURE_DIRTY_CACHE)
 
 /// features that will be implicitly enabled
 #define RBD_FEATURES_IMPLICIT_ENABLE  (RBD_FEATURE_STRIPINGV2  | \
                                        RBD_FEATURE_DATA_POOL   | \
                                        RBD_FEATURE_FAST_DIFF   | \
                                        RBD_FEATURE_OPERATIONS  | \
                                        RBD_FEATURE_MIGRATING   | \
-                                       RBD_FEATURE_NON_PRIMARY)
+                                       RBD_FEATURE_NON_PRIMARY | \
+				       RBD_FEATURE_DIRTY_CACHE)
```

Comment:
Nit: indentation

Reply:
I removed it from RBD_FEATURES_IMPLICIT_ENABLE  as it is not needed.

Reply:
Updated